fetch-utils.js

```diff
@@ -5,16 +5,19 @@ const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export async function getDogs() {
     // from the dogs table, select all items
-
```

Comment:
Great job validating the response you get back to make sure you are able to grab your data